src/shared/ui/Loader/Loader.tsx

```diff
@@ -0,0 +1,18 @@
+import { Component } from 'react';
+import css from './style.module.css';
```

Comment:
by the way more common way is to use `styles` instead of `css`
```suggestion
import styles from './style.module.css';
```
Imagine the situation your app is migrating from css to scss or sass

Reply:
what kind of difference how naming the variable was?
you mean 'style' is more general word (and css - not)?

Reply:
exactly! `styles` is more widely used and generally describes meaning of import